core/arch/arm/kernel/kern.ld.S

```diff
@@ -61,6 +61,13 @@
 #define TEE_TEXT_VA_START       (TEE_RAM_VA_START + \
 					(TEE_LOAD_ADDR - TEE_RAM_START))
 
+/*
+ * Note:
+ * Clang 11 (ld.lld) generates non-relocatable reference when using ROUNDDOWN()
+ * from <util.h>, which does not work with ASLR.
+ */
+#define LD_ROUNDDOWN(x, y) ((x) - ((x) % (y)))
```

Comment:
maybe a tabulation before the macro value

Reply:
A tab would not improve anything here, there is nothing to align with.